app/src/main/java/com/duckduckgo/app/browser/favorites/NewTabLegacyPageView.kt

```diff
@@ -211,10 +208,11 @@ class NewTabLegacyPageView @JvmOverloads constructor(
     }
 
     private fun configureQuickAccessGridLayout(recyclerView: RecyclerView) {
-        val numOfColumns = gridViewColumnCalculator.calculateNumberOfColumns(QUICK_ACCESS_ITEM_MAX_SIZE_DP, QUICK_ACCESS_GRID_MAX_COLUMNS)
+        val gridColumnCalculator = GridColumnCalculator(context)
```

Comment:
Why not injecting the calculator anymore?

Reply:
This class is going away. It’s a static class hidden as a dependency.